src/Document.tsx

```diff
@@ -103,7 +103,10 @@ export const Document = ({ language, children, chunks = [], devEntrypoint }: Pro
         <Scripts />
         {!!entryPoint && <script type="module" src={`/${entryPoint.file}`}></script>}
         {css.map((file) => (
-          <link rel="stylesheet" href={`/${file}`} key={file} />
+          <>
+            <link rel="preload" href={`/${file}`} key={`${file}-preload`} as="stylesheet" />
```

Comment:
Tror det skal være `style` her (https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/link#as)


```suggestion
            <link rel="preload" href={`/${file}`} key={`${file}-preload`} as="style" />
```

Men vet ikke om denne preloaden har noe å si når den ligger rett inntil den faktiske `<link>´-en? 🤔